src/components/tags/tags.module.scss

```
@@ -1,22 +1,32 @@
 @import "/src/css/variables";
 
+.tagsContainer {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+// .tag {
+//   border: 1.5px solid rgba(155, 155, 155, 0.5);
+//   border-radius: 6.49875px;
+//   height: 1.6rem;
+//   margin: 0 1rem 1rem 0;
+//   text-align: center;
+//   padding: 0 1rem;
+//   line-height: 1.7rem;
+//   font-family: $tags-font;
+// }
+
 .tag {
   border: 1.5px solid rgba(155, 155, 155, 0.5);
   border-radius: 6.49875px;
-  height: 1.6rem;
   margin: 0 1rem 1rem 0;
-  text-align: center;
-  padding: 0 1rem;
-  line-height: 1.7rem;
+  text-align: left;
+  padding: 0.3rem 0.3rem;
```

When you set both vertical and horizon axes with the same value, you can simplify it with the value only once and the browser is gonna use it for all axes
```suggestion
  padding: 0.3rem;
```

---

Done